tests/sdk_test/sdk_test.h

```diff
@@ -283,13 +284,20 @@ class MegaChatApiTest :
     bool mCallReceived[NUM_ACCOUNTS];
     bool mCallAnswered[NUM_ACCOUNTS];
     bool mCallDestroyed[NUM_ACCOUNTS];
-    megachat::MegaChatHandle mCallEmisorId[NUM_ACCOUNTS];
+    int mTerminationCode[NUM_ACCOUNTS];
+    bool mTerminationLocal[NUM_ACCOUNTS];
+    megachat::MegaChatHandle mChatIdRingInCall[NUM_ACCOUNTS];
     bool mCallRequestSent[NUM_ACCOUNTS];
-    megachat::MegaChatHandle mCallRequestSentId[NUM_ACCOUNTS];
+    megachat::MegaChatHandle mCallIdRequestSent[NUM_ACCOUNTS];
     megachat::MegaChatHandle mIncomingCallId[NUM_ACCOUNTS];
 
-    megachat::MegaChatHandle mCallId[NUM_ACCOUNTS];
+    megachat::MegaChatHandle mChatIdCallUpdate[NUM_ACCOUNTS];
     megachat::MegaChatHandle mChatIdInProgressCall[NUM_ACCOUNTS];
+
+    const int mInvalidTermCode = 0x7f;
```

Comment:
A user of `MegaChatApi`, as it is this test application, should *not* need to know this internal codes. Please, change this one and the 3 below by termcodes defined at `MegaChatApi` level.